lib/chess_web/templates/layout/app.html.eex

```diff
@@ -14,6 +14,17 @@
   <body>
     <div class="container">
       <header role="banner">
+        <nav role="user">
+          <%= if current_user = Guardian.Plug.current_resource(@conn) do %>
```

Comment:
In Rails I would do this by having a method in `ApplicationController` that defines `@current_user`. It seems Pipelines and Plugs are the way to do that in Phoenix, but I'm not sure on the best implementation.

Reply:
In `Chess.Web`, you can import/make a function into all the views that will make `current_user(@conn)` available in templates and views. Same with `logged_in?(@conn)` or whatever helpers you want.

Reply:
Ah of course, I forgot about views!